.taskcluster.yml

```diff
@@ -206,6 +210,35 @@ tasks:
           owner: mcastelluccio@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-pull-request" && "Train with task cluster:" in pr_description'
+        then:
+          dependencies:
+            - { $eval: as_slugid("docker_build") }
+          taskId: { $eval: as_slugid("train_on_taskcluster") }
+          created: { $fromNow: "" }
+          deadline: { $fromNow: "1 day" }
+          provisionerId: proj-bugbug
+          workerType: compute-smaller
```

Comment:
I wanted to know what would be the most ideal `workerType` for training the models. Training some of the models e.g. `component` are extremely memory intensive.